GitWorkflow/GitWorkflow/Utility.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <SFML/Graphics/Rect.hpp>
+#include <vector>
+
+namespace util
+{
+	const sf::FloatRect& intersection(const sf::FloatRect& rect1, const sf::FloatRect& rect2);
+}
```

Comment:
I DO really like the addition of a utility namespace.

Reply:
I also like it.